src/Compiler/NodeVisitor/AddVarTypesNodeVisitor.php

```diff
@@ -5,12 +5,14 @@
 namespace Efabrica\PHPStanLatte\Compiler\NodeVisitor;
 
 use Efabrica\PHPStanLatte\Compiler\NodeVisitor\Behavior\ActualClassNodeVisitorBehavior;
+use Efabrica\PHPStanLatte\Template\Variable;
```

Comment:
Duplicity, one row under is the same use but aliased. Delete one of them (probably aliasing is no need anymore)

Reply:
I merged it now, cleanup can be done later. Now I have to go to trip with family :)